tests/phpunit/integration/Modules/Analytics_4Test.php

```diff
@@ -901,7 +895,7 @@ function ( $call ) {
 
 		// Ensure the enhanced measurement settings request was made.
 		$this->assertCount( 1, $enhanced_measurement_settings_requests );
-		list( $request ) = array_values( $enhanced_measurement_settings_requests );
+		list($request) = array_values( $enhanced_measurement_settings_requests );
```

Comment:
I will however revert this change because, although the linter hasn't picked it up, we do pad argument lists with spaces pretty much everywhere in the codebase.

```suggestion
		list( $request ) = array_values( $enhanced_measurement_settings_requests );
```